projects/demo/src/modules/components/tooltip/tooltip.component.ts

```diff
@@ -40,6 +42,11 @@ export class ExampleTuiTooltipComponent {
         LESS: example3Less,
     };
 
+    readonly example4: FrontEndExample = {
```

Comment:
We now support promises in examples, so please rewrite it like so:
```ts
readonly example4 = {
    TypeScript: import('!!raw-loader!./examples/4/index.ts'),
    HTML: import('!!raw-loader!./examples/4/index.html'),
};
```
Rewriting the rest of the examples in this file and in your other relevant PRs would be super helpful too! 👍